core/build.gradle

```diff
@@ -146,6 +146,7 @@ processResources.dependsOn ':sparkling-water-assembly-extensions:embeddedAssembl
 
 bench {
     systemProperty "spark.ext.h2o.backend.cluster.mode", detectBackendClusterMode()
+    systemProperty "spark.ext.h2o.rest.api.based.client", "false"
```

Comment:
We first need to switch H2OContext methods asH2OFrame etc to rest api before we can allow client mode in sw core tests